core/config/initializers/07_dragonfly.rb

```diff
@@ -17,6 +17,14 @@
 
   url_format '/media/:job/:name'
   processor :optim, Workarea::ImageOptimProcessor
+
+  response_header 'Cache-Control' do |job, request, headers|
+    if request.path =~ /sitemap/
+      'public, max-age=86400' # 1 day (equal to sitemap generation frequency)
+    else
+      'public, max-age=31536000' # 1 year (Dragonfly default)
```

Comment:
Maybe these times should be configs? Thoughts?

Reply:
i'd lean towards no because the sitemap caching is set to mimic the frequency in which it might change, and the other is a default no one has ever asked or seemed to care about.

Reply:
Yeah fair enough.